### ISSUE TITLE
Auto power on 

### ISSUE THREAD

**Issue by taboca** [apc-io/apc-firefox-os#35]
I wonder if you know how can we make the auto power on for this appliance. I am using it for a dashboard case and after a power outage the system has not started automatically.

**Comment by lechau-apc** [comment]
@taboca We aim to ensure support for features provided by Mozilla's implementation, while disabling non relevant features (e.g. phone-specific) and enabling features specific to our hardware. For the latter, it must be first supported by the under layer system.

@cmchen6585 Assigning this temporarily to you. Please check:
- if this can be supported by the BSP.
- if yes, if this is relevant to support by FF OS.

**Comment by cmchen6585** [comment]
Auto power on is able to enable by remove RTC battery and short PWR_BT pin header (location is on left-down side, top of 2x1 pin header), but when PWR_BT is shorted, power button is abnormal, you need to remove adapter to power off your APC Rock/Paper.

**Comment by lechau-apc** [comment]
So not "programmatically" possible while preserving usual functions, then... @taboca

**Comment by taboca** [comment]
So if I want to deploy this in the enterprise I could create a timer with
the power switch right?

**Comment by cmchen6585** [comment]
Yes! you can have a timer to turn the power adapter on/off, but you can't power off your APC by press power button or click power off in power scheme anymore, that's the limitation.
